README.md

```diff
@@ -76,9 +76,13 @@ Installing nsxiv:
 
     # make install
 
-Please note, that this requires root privileges.
+Installing icons:
+
+	# make -C icon install
```

Comment:
We're currently using 4 spaces for codeblocks in README.
Maybe we should switch to tabs... Anyways that should be it's own PR. For now let's keep things unified.
```suggestion
    # make -C icon install
```